media-source-respec.html

```diff
@@ -3,8 +3,8 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Media Source Extensions™</title>
-    <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove"></script>
-    <!--<script src="respec-w3c-common.js" class="remove"></script>-->
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
```

Comment:
Why is this necessary? Is the deprecation note at https://respec.org/docs/#postProcess outdated such that respec no longer includes jQuery? It seems a bugfix to remove jQuery dependency in the various .js for this spec would be good, regardless. WDYT?

Reply:
Indeed, the deprecation note is now outdated, jQuery was dropped in the new profile, see comment at the beginning of https://github.com/w3c/respec/blob/develop/profiles/w3c-common.js

I reported the need to update the doc in https://github.com/w3c/respec/issues/3228

Regarding dropping jQuery, I agree. I'm proceeding in steps :)

For the time being, `media-source.js` needs jQuery. I'll see how to drop that dependency. Actually, I believe that we can aim at dropping most of `media-source.js` in the end. Cross-references are now pretty well supported in ReSpec, so we should be able to restrict the need to maintain custom code to a bare minimum. Also, relying on ReSpec would allow us to detect problems (such as definitions that other specs don't export) more easily, and to benefit from features such as the context menu that lists references to a given term.

Reply:
All SGTM. Thank you.